Source/ablastr/utils/msg_logger/MsgLogger.cpp

```diff
@@ -211,11 +211,19 @@ MsgWithCounterAndRanks::deserialize (std::vector<char>::const_iterator&& it)
     return MsgWithCounterAndRanks::deserialize(it);
 }
 
-Logger::Logger() :
-    m_rank{amrex::ParallelDescriptor::MyProc()},
-    m_num_procs{amrex::ParallelDescriptor::NProcs()},
-    m_io_rank{amrex::ParallelDescriptor::IOProcessorNumber()}
-{}
+#ifdef AMREX_USE_MPI
+    Logger::Logger() :
+        m_rank{amrex::ParallelDescriptor::MyProc()},
+        m_num_procs{amrex::ParallelDescriptor::NProcs()},
+        m_io_rank{amrex::ParallelDescriptor::IOProcessorNumber()}
+    {}
+#else
+    Logger::Logger() :
+        m_rank{0},
+        m_num_procs{0},
+        m_io_rank{0}
+    {}
+#endif
```

Comment:
Was this code change part of the PR or was it accidentally included?

Reply:
It is actually part of the PR! `AMReX_ParallelDescriptor.H` was probably included transitively even if `AMREX_USE_MPI` was not defined.

Reply:
I see, thanks! So I guess `m_num_procs` is not used at all in the rest of the code, when `AMREX_USE_MPI` is not defined? Is that why we can set `m_num_procs{0}` in this case? I'm asking because intuitively I was expecting `m_num_procs{1}` as minimum value set in the trivial case of one single process.

Reply:
Good observation indeed... on a second thought I think that we can always include `AMReX_ParallelDescriptor.H` in this file, since it should already be able to deal with the case in which `AMREX_USE_MPI` is undefined.